Automattic/wp-calypso
# WIP: Componentize proposed signup header

#### Changes proposed in this Pull Request

* We're trying out a new "masterbar-like" style for the signup flow to save on vertical space and remove distractions as we move toward a new onboarding experience. See #28886 for more information.
* This PR is purely technical; it pulls out the markup/logic and CSS for the new signup header into its own component to clean up the code.

![49097807-92854580-f23b-11e8-95e6-268f04614399](https://user-images.githubusercontent.com/2124984/49233420-f63a7a80-f3c3-11e8-99ad-ed380e4f98a1.png)

#### Testing instructions

* Switch to this PR, navigate to `/start`
* You should see the new signup header styles
* Progress indicator should display the correct numbers
